Description: Use the proper Pari include pari/pari.h
 This is necessary for Pari 2.7 because it now uses multiarch
 include paths.
Author: Tobias Hansen <thansen@debian.org>
Bug-Debian: http://bugs.debian.org/743327

--- a/include/Lcommandline.h
+++ b/include/Lcommandline.h
@@ -39,7 +39,7 @@
 
 #include "Lcommandline_globals.h"      //command line global variables
 #ifdef INCLUDE_PARI
-#include "pari.h"          //for pari's elliptic curve functions
+#include "pari/pari.h"     //for pari's elliptic curve functions
 #undef init                //pari has a '#define init pari_init' which
                            //causes trouble with the stream.h init.
                            //pari also causes trouble with things like abs.
--- a/include/Lcommandline_elliptic.h
+++ b/include/Lcommandline_elliptic.h
@@ -32,7 +32,7 @@
 
 
 #ifdef INCLUDE_PARI
-#include "pari.h"          //for pari's elliptic curve functions 
+#include "pari/pari.h"     //for pari's elliptic curve functions 
 #undef init                //pari has a '#define init pari_init' which
                            //causes trouble with the stream.h init.
                            //pari also causes trouble with things like abs.
